src/zope/app/wsgi/testing.py

```diff
@@ -37,8 +37,8 @@ def __call__(self):
         fn = tempfile.mktemp()
         with open(fn, 'wb') as file:
             file.write(b"Hello\nWorld!\n")
-        file = open(fn, 'rb')
-        return file
+        with open(fn, 'rb') as file:
+            return file.read()
```

Comment:
I'm not sure this is testing what it was testing before. Because the line above writes a constant to the file, this whole function could be simplified to just `return b'Hello\nWorld!\n'`

Reply:
You are right, it now does something completely different. I'll dig into the code to find out where the file should be closed as it seems to be a feature to return a file handle from a view.